pkg/master/master.go

```diff
@@ -188,6 +188,9 @@ func (m *Master) InstallAPIs(c *Config) {
 			Scheme:               api.Scheme,
 			ParameterCodec:       api.ParameterCodec,
 			NegotiatedSerializer: api.Codecs,
+			SubresourceGroupVersionKind: map[string]unversioned.GroupVersionKind{
+				"replicationcontrollers/scale": unversioned.GroupVersion{Group: "autoscaling", Version: "v1", Kind: "Scale"},
```

Comment:
Looks like this is causing a compiler error.  Did you mean `unversioned.GroupVersionKind{Group: "autoscaling", Version: "v1", Kind: "Scale"}`?